### DISCUSSION TITLE
Language change

### DISCUSSION THREAD

**Discussion by TheSlowlySnail** [ahmadfaizalbh/Chatbot#41] [Q&A] [answered]
Hello, I'm currently learning Python so check out Chatbot. Now I've discovered other languages ​​in the source files and am wondering how I can use them? Can you help me with this? Do I have to specify a specific parameter?

**Comment by ahmadfaizalbh** [comment, marked as answer]
You can pass parameter `language` while initiating `Chat` as follows
```python
Chat("main.template", language="de")
```
Or while calling demo

```python 
demo(language="pt-br")
```

## Supported Languages
|       Language              |        Code        |
| --------------------------------- | --------------------- |
| English (default)        | en                    |
| German                       | de                    |
| Portuguese (Brazil)   | pt-br                |
